x-pack/plugins/security_solution/public/common/containers/sourcerer/index.tsx

```diff
@@ -292,7 +292,22 @@ export const useInitSourcerer = (
 };
 
 const LOGS_WILDCARD_INDEX = 'logs-*';
-export const EXCLUDE_ELASTIC_CLOUD_INDEX = '-*elastic-cloud-logs-*';
+export const EXCLUDE_ELASTIC_CLOUD_INDICES = [
+  '-*elastic-cloud-logs-*',
+  '-.ds-logs-elastic_agent*',
```

Comment:
@kevinlog would you be willing to confirm this exclude pattern has no effect on security events?

Reply:
Do we still want to exclude `-.ds-logs-elastic_agent*` and `-.ds-logs-system-*` by default?